Merge packaging & Release 0.0.16

UTsweetyfish opened the pull request:
- **Merge packaging**
- **Bump version to 0.0.16 in archlinux/PKGBUILD and CMakeLists.txt**
- **Declare Breaks and Replaces against libgio-qt (<= 0.0.14)**
- **Drop libgio-qt-doc**
- **Release to unstable**

UTsweetyfish committed:
packaging: merge packaging

- Update debian/changelog to 0.0.16
- Split Qt5/Qt6 library packages: libgio-qt0 & libgio-qt6-0
- Replace deprecated debian/compat with debhelper-compat (= 13)
- Refresh d/control and d/rules
- Drop obsolete .qch documentation installation
  https://github.com/deepin-community/gio-qt/pull/2
- Add proper Multi-Arch, copyright and docs handling

UTsweetyfish committed:
chore: bump version to 0.0.16 in CMakeLists.txt and archlinux/PKGBUILD

Synchronize the hard-coded version strings:
- CMakeLists.txt: update PROJECT_VERSION default from 0.0.1 to 0.0.16
- archlinux/PKGBUILD: update pkgver from 0.0.14 to 0.0.16

UTsweetyfish committed:
fix: declare Breaks/Replaces against libgio-qt <= 0.0.14

Added Breaks and Replaces relations against the legacy libgio-qt binary
packages (version 0.0.14 and earlier) to all three binary packages
(libgio-qt-dev, libgio-qt5-0, libgio-qt6-0).

UTsweetyfish committed:
chore: Release version 0.0.16 to unstable

Update changelog

BLumia commented on CMakeLists.txt:
```diff
@@ -52,7 +52,7 @@ if (BUILD_DOCS)
     set (DOXYGEN_QHG_LOCATION "qhelpgenerator")
     set (DOXYGEN_QHP_NAMESPACE "org.deepin.gio-qt")
-    set (DOXYGEN_QCH_FILE "../gio-qt.qch")
+    #set (DOXYGEN_QCH_FILE "../gio-qt.qch")
```
这个是为啥去掉的

UTsweetyfish replied:
不知道，我 as-is 照搬的
https://github.com/deepin-community/gio-qt/pull/2

UTsweetyfish replied:
可以留着？要留着吗？

BLumia replied:
这个和下面那个应该是出qch文档（-doc包）用的，现在我看-doc包已经去掉了，相当于不会生成文档。。

BLumia reviewed (commented):
四个commit要原样保留吗？如果要的话，commit格式不合规，不要的话我 squash 合并的时候可以修改掉。

UTsweetyfish committed:
chore: Drop libgio-qt-doc package

Remove the libgio-qt-doc binary package from the build. It's already an
empty package.

UTsweetyfish commented:
> 四个commit要原样保留吗？如果要的话，commit格式不合规，不要的话我 squash 合并的时候可以修改掉。

我看看改下 commit 格式？

BLumia commented:
> > 四个commit要原样保留吗？如果要的话，commit格式不合规，不要的话我 squash 合并的时候可以修改掉。
> 
> 我看看改下 commit 格式？

要保留的话，改吧。

```
type: short description

detail desciption

Log:
```

这种格式。commitlint 也是按这个检查的。

UTsweetyfish commented:
qch 的变更要 revert 掉吗？

BLumia reviewed (approved):
文档回头有人有空再说吧。。

UTsweetyfish commented:
/integrate

Zeno-sole commented:
本次集成关闭，后续由成都方面发起集成
